src/main/scala/com/atomist/source/git/GitHubServices.scala

```diff
@@ -576,6 +576,12 @@ case class GitHubServices(oAuthToken: String, apiUrl: Option[String] = None)
                                             content: Option[ReactionContent] = None): Seq[Reaction] =
     listReactions(s"$api/repos/$owner/$repo/pulls/comments/$id/reactions", content)
 
+  def listIssueEvents(repo: String,
+                      owner: String,
+                      number: Int,
+                      params: Map[String, String] = Map("per_page" -> "100")): Seq[IssueEvent] =
+    paginateResults[IssueEvent](s"$api/repos/$owner/$repo/issues/$number/events", params)
```

Comment:
It's better to retry than not, but I'm not sure how that would work for pagination. Since it never has retried, I don't think it fair to prevent these from being merged if it is not because `paginateResults` is not retried.